jdbc/spring-cloud-starter-stream-sink-jdbc/src/main/resources/META-INF/spring-configuration-metadata-whitelist.properties

```diff
@@ -1 +1,8 @@
 configuration-properties.classes=org.springframework.cloud.stream.app.jdbc.sink.JdbcSinkProperties
+configuration-properties.names=\
+spring.datasource.url,\
+spring.datasource.driver-class-name,\
+spring.datasource.username,\
+spring.datasource.password,\
+spring.datasource.init-sql,\
+spring.datasource.initialize
```

Comment:
Does S3 really need a similar fix?
This particular case is really because there are 5 classes that contribute to the same `spring.datasource` group, and that we don't wont to add those 5 classes (different implementations of a jdbc pool).

In the common case, listing classes should work ok.

Anyway, here is the final result:
```
dataflow:>app info --id source:jdbc
Information about source application 'jdbc':
Resource URI: maven://org.springframework.cloud.stream.app:jdbc-source-rabbit:1.0.0.BUILD-SNAPSHOT
╔══════════════════════════════╤══════════════════════════════╤══════════════════════════════╤══════════════════════════════╗
║         Option Name          │         Description          │           Default            │             Type             ║
╠══════════════════════════════╪══════════════════════════════╪══════════════════════════════╪══════════════════════════════╣
║trigger.time-unit             │The TimeUnit to apply to delay│<none>                        │java.util.concurrent.TimeUnit ║
║                              │values.                       │                              │                              ║
║trigger.fixed-delay           │Fixed delay for periodic      │1                             │java.lang.Integer             ║
║                              │triggers.                     │                              │                              ║
║trigger.cron                  │Cron expression value for the │<none>                        │java.lang.String              ║
║                              │Cron Trigger.                 │                              │                              ║
║trigger.initial-delay         │Initial delay for periodic    │0                             │java.lang.Integer             ║
║                              │triggers.                     │                              │                              ║
║trigger.max-messages          │Maximum messages per poll, -1 │1                             │java.lang.Long                ║
║                              │means infinity.               │                              │                              ║
║trigger.date-format           │Format for the date value.    │<none>                        │java.lang.String              ║
║jdbc.max-rows-per-poll        │Max numbers of rows to process│0                             │java.lang.Integer             ║
║                              │for each poll.                │                              │                              ║
║jdbc.query                    │The query to use to select    │<none>                        │java.lang.String              ║
║                              │data.                         │                              │                              ║
║jdbc.update                   │An SQL update statement to    │<none>                        │java.lang.String              ║
║                              │execute for marking polled    │                              │                              ║
║                              │messages as 'seen'.           │                              │                              ║
║jdbc.split                    │Whether to split the SQL      │true                          │java.lang.Boolean             ║
║                              │result as individual messages.│                              │                              ║
║spring.datasource.password    │<unknown>                     │<none>                        │java.lang.String              ║
║spring.datasource.url         │<unknown>                     │<none>                        │java.lang.String              ║
║spring.datasource.initialize  │Populate the database using   │true                          │java.lang.Boolean             ║
║                              │'data.sql'.                   │                              │                              ║
║spring.datasource.driver-class│<unknown>                     │<none>                        │java.lang.String              ║
║-name                         │                              │                              │                              ║
║spring.datasource.init-sql    │<unknown>                     │<none>                        │java.lang.String              ║
║spring.datasource.username    │<unknown>                     │<none>                        │java.lang.String              ║
╚══════════════════════════════╧══════════════════════════════╧══════════════════════════════╧══════════════════════════════╝
```